frontstage/templates/account/account-contact-detail-change.html

```diff
@@ -149,7 +149,7 @@ <h1 class="u-fs-xl">Change your contact details</h1>
             "submitType": "timer"
             })
         }}
-        <a href="{{ url_for('account_bp.get_account', tag='todo') }}"
+        <a href="{{ url_for('account_bp.get_account') }}"
       role="button" class="btn btn--secondary" id="btn-upload-ci-cancel"><span class="btn__inner">Cancel</span></a>
```

Comment:
I know you didn't write this (I think it was me who did sorry...), but can you change the id of this button to something more sensible.

Reply:
looks like I did write it.. will change it